src/Microsoft.Extensions.Localization.Abstractions/LocalizedString.cs

```diff
@@ -64,6 +77,11 @@ public LocalizedString(string name, string value, bool resourceNotFound)
         public bool ResourceNotFound { get; }
 
         /// <summary>
+        /// The location which was searched for a locatization value.
+        /// </summary>
+        public string LocationSearched { get; }
```

Comment:
I didn't said its the behavior of `ResourceManagerStringLocalizer` 😃 , even the intent of making this change was to help diagnosing given that you have a localized string. I'm still disagree to add the `LocationSearched` or `SearchedLocation` to the `LocalizedString` itself, if we think deeply for a minute the `ResourceNotFound` is suitable here, because we may find the resource or might not, but the `SearchedLocation` is something generic for all the resources

I will try to create a branch in my localization repo to prove the concept